swagger.yaml

```diff
@@ -30,6 +26,22 @@ paths:
           schema:
             $ref: '#/definitions/Info'
 
+  /departments:
+    get:
+      summary: List departments
+      description: Return org code and org description (departments) for a business center
+      parameters:
+        - in: query
```

Comment:
I was thinking about allowing just one business center at a time to make the results a bit easier to consume.